.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Discord Community Support
+    url: https://discord.gg/getpieces
+    about: You can ask and answer questions here.
+  - name: Documentation
+    url: https://docs.pieces.app/
+    about: Pieces documentation.
```

Comment:
@Sophyia7 we should add a little more to this description here

Reply:
@jordan-pieces I adjusted it, and I also added our YT page.